src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/build/buildMultiTargeting/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.targets

```diff
@@ -0,0 +1,18 @@
+<!--
+Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Remove the copyright header. Per my last conversation about this with @Eilon, we don't put the copyright header into .targets/props files that ship in nuget packages.